_data/links.yml

```diff
@@ -0,0 +1,252 @@
+- category:  Engineering Blogs
+  side_bar_name: Engineering Blogs
+  categoryDesc: 
+  links: 
+  - title: Booking.com B.V.
+    links:
+      - linkName:  Booking.com dev blog 
+        url: https://blog.booking.com/
+        linkDesc: "  "
```

Comment:
Can we avoid so many empty linkDesc ?

Reply:
some links contain some text in addition to link-text. "linkDesc" is included to give that flexibility. assigning empty string does no harm.

Reply:
brings us to some sort of trade-off here

Reply:
I personally vote for having "linkDesc"